XCUITests/FxScreenGraph.swift

```diff
@@ -99,11 +100,18 @@ func createScreenGraph(_ app: XCUIApplication, url: String = "https://www.mozill
             }
 
             scene.tap(startBrowsingButton, to: NewTabScreen)
+            scene.tap(app.buttons["Sign in to Firefox"], to: FxSyncLogin)
         }
 
         i += 1
     }
 
+    map.createScene(FxSyncLogin) { scene in
+        scene.backAction = {
+            app.buttons["Cancel"].tap()
```

Comment:
Looking further down, is this actually a `backAction`? 

If you press *Cancel*, then it takes you back to the tab, not to tour?

I'm actually not sure what the correct answer here is; I can identify a number of ways of getting to `FxSyncLogin`, and the top left button does something different each time:

 * `FirstRun` -> Tour -> `FxSyncLogin`, press *Cancel* takes you to `NewTabScreen`.
 * `BrowserTab` -> `SettingsScreen` -> Show Tour -> Tour -> `FxSyncLogin`, pressing *Cancel* takes you back to the `BrowserTab`.
 * `TabTray` -> `SettingsScreen` -> Show Tour -> Tour -> `FxSyncLogin`, pressing *Cancel* takes you back to the `TabTray`.
 * `TabTray` -> `SettingsScreen` -> Sign in to Firefox -> `FxSyncLogin`, there is no *Cancel* button, but there is a *Settings* back button, which takes you back to `SettingsScreen`.

There are three approaches here, neither of which I like: 

1. File a bug against the FxSyncLogin screen that the Cancel button should behave like a back button, so go back to the Tour, if accessed from the tour. @tecgirl should probably have a look.
2. file a bug against `ScreenGraph` such that "Enable ScreenGraph model multi level navigation"